cosmosdb/types.bal

```diff
@@ -354,12 +359,13 @@ public type ResourceReadOptions record {|
     string sessionToken?;
 |};
 
-# Represent the optional parameters which can be passed to the function when querying containers.
+# Optional parameters which can be passed to the function when querying containers.
 # 
-# + consistancyLevel - The consistency level override. Allowed values are `Strong`, `Bounded`, `Session` or `Eventual`.
-#                      The override must be the same or weaker than the account’s configured consistency level.
+# + consistancyLevel - The consistency level override
+#                    - Allowed values are `Strong`, `Bounded`, `Session` or `Eventual`.
+#                    - The override must be the same or weaker than the account’s configured consistency level.
 # + sessionToken - Echo the latest read value of `sessionToken` to acquire session level consistency 
-# + enableCrossPartition - Boolean value specifying whether to allow cross partitioning. Default is `true` where, 
+# + enableCrossPartition - Boolean value specifying whether to allow cross partitioning <br/> Default is `true` where, 
 #                          it allows to query across all logical partitions.
 # + partitionKey - Optional. The value of partition key field of the container.
```

Comment:
Suggestion: "The value of the **Partition Key** field of the container. This is an optional parameter."